ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/PackageConfigurationFactory.kt

```diff
@@ -0,0 +1,48 @@
+package com.revenuecat.purchases.ui.revenuecatui.data.processed
+
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.paywalls.PaywallData
+import java.util.Locale
+
+internal object PackageConfigurationFactory {
+    @Suppress("LongParameterList")
+    fun createPackageConfiguration(
+        packages: List<Package>,
+        activelySubscribedProductIdentifiers: Set<String>,
+        filter: List<String>,
+        default: String?,
+        localization: PaywallData.LocalizedConfiguration,
+        configurationType: PackageConfigurationType,
+        locale: Locale,
+    ): TemplateConfiguration.PackageConfiguration {
+        val packagesById = packages.associateBy { it.identifier }
+        val filteredRCPackages = filter.mapNotNull { packagesById[it] }
```

Comment:
👍🏻  I came here to check because the previous implementation was failing if a package wasn't found. It should instead ignore it, but maybe it can log something?